### PR TITLE
[YUNIKORN-1571] cleanup old plugin deployment examples

### PR DESCRIPTION
**Pull request by 0yukali0**
### What is this PR for?
Removing unused config in plugin.yaml example

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring
* [x] - Tasks 

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1571

### Screenshots (if appropriate)
image

image

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: deployments/scheduler/plugin.yaml [hunk 1]
```diff
@@ -96,8 +97,6 @@ spec:
               cpu: 4
               memory: 2Gi
           volumeMounts:
-            - mountPath: /etc/kubernetes
-              name: scheduler-config
             - mountPath: /etc/yunikorn
               name: quotamanager-config
```
Comment (wilfred-s) [inline deployments/scheduler/plugin.yaml:101]: With the new configuration v2 do we need these?
I could not find any mounting beside the secrets for the admission controller in the current helm charts.

@craigcondit

Comment (0yukali0) [reply deployments/scheduler/plugin.yaml:101]: Hi @wilfred-s , you are right and I will remove this in next commit.
After I removed the quatamanager-config, latest yunikorn will use the default configuration here.
image

Comment (wilfred-s) [reply deployments/scheduler/plugin.yaml:101]: That looks correct, let's see if we have got this all correct and get a review from Craig.

**Review by wilfred-s — APPROVED** [review]
LGTM

**Pull request closed without merging**